.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,28 @@
+name: golangci-lint
+on:
+  push:
+    tags:
+      - v*
```

Comment:
I think we may skip linters on tag branches. This is required in PRs and may be on master in case of direct commits.